src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -26,7 +26,7 @@
     // Miscellaneous
     "confirmCloseAllTabs": true,
     "startOnUserLogin": false,
-    "theme": "system",
+    "theme": "dark",

```

Comment:
This may cause some people to snap

Reply:
it was definitely a 🌶️ take, for sure. @cinnamon-msft suggested it originally.

Reply:
I think it's much betterto simply leave system as default. If a user wants dark (or light) no matter what, they'll set it explicitly. It's the same thing with Edge. It's system by default as well. Since we're following Edge's example for the theme, this should probably also match.

Reply:
I think the interesting thing that we have today is that the color scheme is dark by default, but our window theme is set to system. System theme in Windows 11 is defaulted to light unless changed by the user. Now, we have a conflict between the theme and color scheme in Terminal.

I think our options become, make the color scheme match "default" and set it to a light color scheme if the system theme is light, or manually match the theme to the color scheme by setting it to dark.

Given that Terminal has historically had a black background with its Campbell color scheme, for a consistent UI experience, I'm voting to change the window theme to be dark by default as well.

Reply:
I agree. People who don't like it can switch it back -- they are likely accustomed to applications acting strange in light theme ;P

Reply:
lmao wassup reddit 😜